hack/olm-registry/olm-artifacts-template-hive.yaml

```diff
@@ -0,0 +1,268 @@
+apiVersion: v1
```

Comment:
Is there a reason for the new file instead of adding a SSS to the original template? I don't really have a reason for putting it in the same file other than "that's what the other operators do", so mostly just curious.

Reply:
I learned that the olm-artifacts template is targeted in app-interface. Since that's the case I'm thinking more about a `olm-artifacts-template-fedramp.yaml` - what do you think?

Reply:
Sounds just fine